src/core/file_sys/bis_factory.cpp

```diff
@@ -12,6 +12,10 @@
 
 namespace FileSys {
 
+constexpr u64 NAND_USER_SIZE = 0x680000000ULL;  // 26624 MiB
```

Comment:
```suggestion
constexpr u64 NAND_USER_SIZE = 0x680000000;  // 26624 MiB
```

doesn't need to be included, given it's an integer constant being assigned to a u64. This will always be typed correctly, given there's no ambiguity. I realize the previous code did that sort of prefixing, but even then it wasn't necessary either.